test/utilities/StyleMocking.js

```diff
@@ -0,0 +1,62 @@
+import assert from 'assert'
+
+// These methods are useful in tests which verify changes a method makes to element 'style'
+// properties.
+
+/**
+ * An object whose properties are taken to be style property names,
+ * and whose property values are taken to be style property values.
+ * @typedef {Object} StylesHash
```

Comment:
This syntax is neat (and new to me)! It would be even cooler if we could represent a map. I looked in the docs but it wasn't quite clear to me.

http://usejsdoc.org/tags-type.html
http://usejsdoc.org/tags-typedef.html